terraform/base/route53.tf

```diff
@@ -393,13 +457,7 @@ resource "aws_route53_record" "heroku-uplift-shipit-cname-test" {
 ############################
 
 variable "cloudfront_alias" {
```

Comment:
cloudfront_alias and cloudfront_alias_domain are now unused variables and should be removed.  And so should resource "aws_route53_record" "cloudfront-alias"